src/serverMain/resources/viewQR/styles.css

```diff
@@ -27,16 +27,38 @@ header > h2 {
   font-size: 32px;
 }
 
+.small-pages-container {
+  /* width must be 1mm less than A4 because of rounding errors that would otherwise leave empty pages */
+  height: 297mm;
+  width: 209mm;
+  page-break-after: always;
+  transform: translateX(209mm) rotate(90deg);
```

Comment:
btw, very nicely done in css :)

Reply:
Thanks, I think 80% of the time I spent wrestling css weirdness ^^